test/requestinterception.spec.ts

```diff
@@ -52,13 +52,17 @@ describe('request interception', function () {
       expect(response.ok()).toBe(true);
       expect(response.remoteAddress().port).toBe(server.PORT);
     });
+    // @see https://github.com/puppeteer/puppeteer/pull/3105
     it('should work when POST is redirected with 302', async () => {
       const { page, server } = getTestState();
 
       server.setRedirect('/rredirect', '/empty.html');
       await page.goto(server.EMPTY_PAGE);
       await page.setRequestInterception(true);
-      page.on('request', (request) => request.continue());
+      page.on('request', (request) => {
+        console.log(request.method(), request.url());
```

Comment:
why are console.log messages needed?

Reply:
Good catch, thanks! I thought I had already removed it.